web/app.fluidity.money/app/root.tsx

```diff
@@ -235,15 +237,41 @@ function App() {
     }
   }, []);
 
+  const { showExperiment } = useContext(SplitContext);
+
   const [splitUser, setSplitUser] = useState(splitUserKey);
 
+  // show the user a banner if they're on our "bother"
+  // list for being interesting
+
+  const [showTargetedBanner, setShowTargetedBanner] = useState(false);
+
+  useEffect(() => {
+    if (!showExperiment("target-banner")) {
+      return;
+    }
+
+    const _targetedBannerSeen = localStorage.getItem("targetedBanner");
+    if (!_targetedBannerSeen) {
+      setShowTargetedBanner(false);
```

Comment:
assuming this should be `setShowTargetedBanner(true)`?

Reply:
True, and it should be a lot cleaner too